src/components/BulkOverrideDialog/BulkOverrideInfo.js

```diff
@@ -143,13 +143,15 @@ class BulkOverrideInfo extends React.Component {
         item: {
           barcode,
         },
+        checkoutServicePointId,
```

Comment:
Is it loan's check-out service point or currently chosen service point?

Reply:
Good catch. Fixed.